media/Config/index.js

```diff
@@ -0,0 +1,207 @@
+    { optionName: "fuse_activation_function", optionValue: false },
+    { optionName: "fuse_instnorm ", optionValue: false },
+    {
+      optionName: "replace_cw_mul_add_with_depthwise_conv",
+      optionValue: false,
+    },
+    { optionName: "remove_fakequant", optionValue: false },
+    { optionName: "remove_quantdequant", optionValue: false },
+    { optionName: "remove_redundant_reshape", optionValue: false },
+    { optionName: "remove_redundant_transpose", optionValue: false },
+    { optionName: "remove_unnecessary_reshape", optionValue: false },
+    { optionName: "remove_unnecessary_slice", optionValue: false },
+    { optionName: "remove_unnecessary_strided_slice", optionValue: false },
+    { optionName: "remove_unnecessary_split", optionValue: false },
+    { optionName: "resolve_customop_add", optionValue: false },
+    { optionName: "resolve_customop_batchmatmul", optionValue: false },
+    { optionName: "resolve_customop_matmul", optionValue: false },
+    { optionName: "resolve_customop_max_pool_with_argmax", optionValue: false },
+    { optionName: "shuffle_weight_to_16x1float32", optionValue: false },
+    { optionName: "substitute_pack_to_reshape", optionValue: false },
+    { optionName: "substitute_squeeze_to_reshape", optionValue: false },
+    { optionName: "substitute_strided_slice_to_reshape", optionValue: false },
+    { optionName: "substitute_transpose_to_reshape", optionValue: false },
+    { optionName: "transform_min_max_to_relu6", optionValue: false },
+    { optionName: "transform_min_relu_to_relu6", optionValue: false },
+  ],
+};
+
+const oneQuantize = {
+  type: "one-quantize",
+  use: true,
+  options: [
+    { optionName: "input_path", optionValue: "" },
+    { optionName: "output_path", optionValue: "" },
+    { optionName: "input_data", optionValue: "" },
+    { optionName: "input_data_format", optionValue: "" },
+    { optionName: "input_dtype", optionValue: "" },
+    { optionName: "quantized_dtype", optionValue: "" },
+    { optionName: "granularity", optionValue: "" },
+    { optionName: "min_percentile", optionValue: "" },
+    { optionName: "min_percentile", optionValue: "" },
+    { optionName: "max_percentile", optionValue: "" },
+    { optionName: "mode", optionValue: "" },
+    { optionName: "generate_profile_data", optionValue: false },
+  ],
+};
+
+const onePack = {
+  type: "one-pack",
+  use: true,
+  options: [
+    { optionName: "input_path", optionValue: "" },
+    { optionName: "output_path", optionValue: "" },
+  ],
+};
+
+const oneCodegen = {
+  type: "one-codegen",
+  use: false,
+  options: [
+    { optionName: "backend", optionValue: "" },
+    { optionName: "command", optionValue: "" },
+  ],
+};
+
+const oneProfile = {
+  type: "one-profile",
+  use: false,
+  options: [
+    { optionName: "backend", optionValue: "" },
+    { optionName: "command", optionValue: "" },
+  ],
+};
+
+// this is entire options for ONE compile
```

Comment:
`these are entire options for ONE compile` ?

Reply:
okay. Thank you!